tests/performance/scripts/runperf.sh

```diff
@@ -35,10 +35,11 @@ cat >config.json<<END
         "node_num": 10,
         "imagerepo": "kubeedge",
         "k8smasterforprovisionedgenodes": "$K8SMasterForProvisionEdgeNodes",
-        "cloudimageurl": "kubeedge/edgecontroller-test:v2.1",
-        "edgeimageurl": "kubeedge/edgecore-test:v2.1",
+        "cloudimageurl": "pavan187/cloudcore:v2.2",
```

Comment:
we'd better to upload to kubeedge, but we can update later when we have v0.3 version:)